### ISSUE TITLE
Images not getting fixed or even loading (in IE6) when brought in from jQuery

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#69]
```
What steps will reproduce the problem?
1. Set up a jQuery file to allow a transparent png background image to be
swapped in when an anchor tag is clicked.

Example of jQuery I am attempting to use:
$("anchor_name").css("background", "url(link to replacement image)");

To put much detailed explanation here would be long and difficult to write
such that it conveys the message well - I have attached a zip file of the
problem which is also visible at http://www.damnneardoomed.com/jq/index.htm

What is the expected output? What do you see instead?

The expectation here is that when "OEM/Consumer," "Electronics" or
"Medical" is clicked, the content in the below box is altered and all three
buttons change to reflect the new content. Instead ( again, IE6 only ), the
new image is not loaded and the background of the current image turns to
that of a non-fixed transparent png. I imagine this is somehow the script
prohibiting new images from being loaded, but the browser still accounts
for the change, so it displays the botched alpha.

That probably doesn't make too much sense, but if you preview the problem
it will, probably, make perfect sense.


What version of the product are you using? On what operating system?

2.0(beta3) - problem exists when using any combination of IE 7/8 and both
or either ie7-squish/ie7-recalc

Running XP SP2, using Multiple IE to emulate IE 6 behavior - actual IE 6
user confirms issue exists in MS release.


Please provide any additional information below.

I attempted to load the images as inline content to see if they might
appear since they had already loaded, but no such luck.

If anything was unclear, please let me know and I will do my best to
rephrase it.
```

Original issue reported on code.google.com by `eric.dec...@gmail.com` on 22 Apr 2008 at 5:38

Attachments:
- [jq.zip](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-69/comment-0/jq.zip)

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in the trunk:

http://ie7-js.googlecode.com/svn/trunk/lib/
```

Original comment by `dean.edw...@gmail.com` on 22 Apr 2008 at 7:21
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
I downloaded and replaced my files with the files from that link and the problem
still occurs. Any ideas?
```

Original comment by `eric.dec...@gmail.com` on 22 Apr 2008 at 7:34

**Comment by GoogleCodeExporter** [comment]
```
I'll need to look into it.
```

Original comment by `dean.edw...@gmail.com` on 24 Apr 2008 at 4:41
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Thanks - no rush though, I've fixed the problem by using non-transparent PNGs.
```

Original comment by `eric.dec...@gmail.com` on 24 Apr 2008 at 4:45

**Comment by GoogleCodeExporter** [comment]
```
I'm going to close this bug and mark it as Invalid. I'm not sure if it is fixed 
or  
not. The provided test case was too complicated. I don't have time to debug 
entire 
sites. If the problem is not fixed in version 2.1. Then resubmit this bug with 
a 
simpler test case.
```

Original comment by `dean.edw...@gmail.com` on 14 Feb 2010 at 12:45
- Changed state: **Invalid**
